types/rest/rest.go

```diff
@@ -442,3 +445,23 @@ func GetRequest(url string) ([]byte, error) {
 
 	return body, nil
 }
+
+// PostRequest defines a wrapper around an HTTP POST request with a provided URL and data.
+// An error is returned if the request or reading the body fails.
+func PostRequest(url string, contentType string, data []byte) ([]byte, error) {
+	res, err := http.Post(url, contentType, bytes.NewBuffer(data))
```

Comment:
I'm not sure how to handle the gosec issue. Consider adding a `nolint` if you cannot figure it out.

Reply:
https://securego.io/docs/rules/g107.html I will use nolint, basically is to avoid passing vars to it if they come from untrusted source which should not be the case.